benchmark/perf_hooks/performance-mark.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common');
+const {
+  PerformanceObserver,
+  performance
+} = require('perf_hooks');
+
+const bench = common.createBenchmark(main, {
+  n: [1e6],
+  buffered: ['true', 'false'],
+  mark: ['true', 'false']
+});
+
+function marked(n, buffered) {
+  bench.start();
+  const obs = new PerformanceObserver(() => {});
+  obs.observe({ entryTypes: ['mark', 'measure'], buffered });
+  performance.mark('A');
+  for (let i = 0; i < n; i++) {
+    performance.mark('B');
+    performance.measure('A to B', 'A', 'B');
+  }
+  bench.end(n);
+}
+
+function unmarked(n) {
+  bench.start();
+  for (let i = 0; i < n; i++) {
+    // Do nothing.
+  }
+  bench.end(n);
+}
+
+function main({ n, buffered, mark }) {
+  if (mark === 'true') marked(n, buffered === 'true');
+  else unmarked(n);
```

Comment:
This is going to run for each other matrix entry. In this case, it's run twice. It is also difficult to compare the result with the other result, since we mostly compare the percentages, not the actual raw numbers. Should we really add this "no-op" benchmark?